README.md

```diff
@@ -4,9 +4,154 @@ Font Asset Generator based on OpenLL Specifications
+```
+
+Create an atlas for the Arial font containing only the glyphs 'a', 'b' and 'c', and write it to `atlas.png`:
+```shell
+llassetgen-cmd atlas --fontname Arial --glyph abc atlas.png
+```
+
+Create an atlas containing all printable ASCII glyphs, with a font size of 256 pixels, then use the Parabola Envelope algorithm to apply a distance transform. Write the resulting distance field to `atlas.png` and generate an FNT file, which is written to `atlas.fnt`:
+```shell
+llassetgen-cmd atlas --ascii --fontsize 256 --distfield parabola --fontname Arial atlas.png --fnt
+```
+
+Since a distance field creates a "glow" around every glyph, add 20 pixels of padding around each glyph in the atlas to create the necessary space. To improve the final rendering quality of the font, apply a 4x downsampling to every glyph:
+```shell
+llassetgen-cmd atlas --padding 20 --downsampling 4 --ascii --distfield parabola --fontname Arial atlas.png
+```
+
+### Rendering
+Additionally to the CLI, you can use the GUI-application `llassetgen-rendering`. It offers a preview of the rendering using the calculated distance field. Using the GUI, you can change all parameters and see their direct impact on the final image.
+
+`llassetgen-rendering` uses the fragment-shader as in [OpenLL](http://openll.org/) for Super Sampling. Rendering parameters are:
+
+<dl>
+  <dt>Super Sampling</dt><dd>Choose the type of Super Sampling.</dd>
+  <dt>Threshold</dt><dd>The default value is 0.5, since the Distance Transform considers 0.5 as the contour. Choosing a smaller value renders the glyphs bold; a larger value renders them thinner, until they completely disappear. A smaller value than 0.3 is clamped to 0.3, because such fragments are dicarded in fragment shader for performance reasons.</dd>
+  <dt>Switch Rendering</dt><dd>You can toggle between rendering the calculated distance field or rendering the resulting glyphs when using the calculated distance field.</dd>
+</dl>
+
+
+## Details on Algorithms and Parameters
+
+<dl>
+  <dt>Font Name</dt><dd>The font is loaded from the local machine using the given font name, e.g. Arial, Verdana</dd>
+  <dt>(Original) Font Size</dt><dd>This size is used to pre-render the glyphs before the Distance Transform is applied. A higher value results in smoother fonts (high resolution font), but the Distance Transform performs slower. We encourage larger font size, as the Distance Field is only generated once.</dd>
+  <dt>Padding</dt><dd>Space that is added around the glyphs. Gives more space for dynamic range of distance transformed glyphs. That means, the glyphs get 'larger' and thus need more space to not be cut off.</dd>
+  <dt>Downsampling</dt><dd>Distance Fields are meant to be downsampled. We offer different downsampling types and also different scaling factors.</dd>
+  <dt>Dynamic Range</dt><dd>The Distance Transform calculates values, that need to be clamped in order to generate a PNG. Choose the min and max values.</dd>
```

Comment:
Although this is the most objectively correct description, the effect of the dynamic range on the distance field is difficult to understand. In the help text of `llassetgen-cmd` I described it like this, maybe you could write something similar?

>Takes two values, BLACK and WHITE. A lower black value will make the distance fields wider; a lower white value will make the distance fields brighter. In most cases, the black value should be lower than the white value. However, swapping the black and white value will invert the colors of the atlas

Reply:
You are also assigned on this. Just correct it :) I like your description.

Reply:
Nevermind.